### PR TITLE
Fix https://github.com/aviatesk/JET.jl/issues/267

### PR DESCRIPTION
**Pull request by goerch**
With this change OP's test shows the same result as 

``` 
import Pkg

Pkg.status()
```

### CODE REVIEW

**Comment by aviatesk** [comment]
Maybe can you add a simple test in https://github.com/aviatesk/JET.jl/blob/master/test/test_misc.jl like:
```julia
@test to_simple_module_usages(:(import Pkg as P)) == Expr[:(import Pkg as P)]
```

File: test/toplevel/test_virtualprocess.jl [hunk 1]
```diff
@@ -86,6 +86,19 @@ end
     end
 end
 
+@testset "test to_simple_module_usages" begin
+    @test JET.to_simple_module_usages(:(using A, B; export a, b)) ==
+        Expr[quote using A, B end, quote export a, b end]
```
Comment (aviatesk) [inline test/toplevel/test_virtualprocess.jl:91]:
```suggestion
    @test JET.to_simple_module_usages(:(using A, B)) ==
        Expr[:(using A), :(using B)]
    @test JET.to_simple_module_usages(:(export a, b)) ==
        Expr[:(export a), :(export b)]
```
`to_simple_module_usages` isn't supposed to take `:block` expression.

Comment (goerch) [reply test/toplevel/test_virtualprocess.jl:91]: The comment in `to_simple_module_usages` 

```
    if length(x.args) != 1
        # using A, B, export a, b
        return Expr.(x.head, x.args)
    else
```

suggested something like that?

File: test/toplevel/test_virtualprocess.jl [hunk 2]
```diff
@@ -86,6 +86,19 @@ end
...
+    @test JET.to_simple_module_usages(:(export a)) ==
+        Expr[:(export a)]
+    @test JET.to_simple_module_usages(:(import Pkg as P)) ==
+        Expr[:(import Pkg as P)]
+    @test JET.to_simple_module_usages(:(using A)) ==
+        Expr[:(using A)]
+    @test JET.to_simple_module_usages(:(using A: sym1, sym2)) ==
+        Expr[:(using A: sym1), :(using A: sym2)]
+end
```
Comment (aviatesk) [inline test/toplevel/test_virtualprocess.jl:100]:
```suggestion
    @test JET.to_simple_module_usages(:(import Pkg: status as s, gc as g)) ==
        Expr[:(import Pkg: status as s), :(import Pkg: gc as g)]
end
```
Thanks for writing up this test set :)

File: test/toplevel/test_virtualprocess.jl (hunk 1 above)
Comment (aviatesk) [reply test/toplevel/test_virtualprocess.jl:91]: To elaborate `# using A, B, export a, b`, this code is supposed to handle patterns like `using A, B` or `export a, b`.

Comment (goerch) [reply test/toplevel/test_virtualprocess.jl:91]: Thanks!

**Pull request merged**